core/src/logging.rs

```diff
@@ -32,14 +32,18 @@ pub fn init(filter: impl AsRef<str>) -> (Logger, GlobalLoggerGuard) {
 }
 
 /// Sets a panic hook so panic information is written with the log facilities
-/// instead of directly to STDERR.
+/// in addition to the default panic printer.
 fn set_panic_hook() {
-    fn hook(info: &PanicInfo) {
+    let default_hook = panic::take_hook();
+    let hook = move |info: &PanicInfo| {
         let thread = thread::current();
         let thread_name = thread.name().unwrap_or("<unnamed>");
 
+        // It is not possible for our custom hook to print a full backtrace on stable rust. To not
+        // lose this information we call the default panic handler which prints the full backtrace.
+        default_hook(info);
         log::error!("thread '{}' {}", thread_name, info);
```

Comment:
Maybe we should log first? This way I imagine that on Kibana the parser will attach the panic message the the error log document.

Reply:
I don't think that guarantees anything because we print log messages at some point later in time in another thread. Maybe we could print a fake log message prefix / separator before the panic though so it's treated as one message in kibana.

Reply:
> I don't think that guarantees anything because we print log messages at some point later in time in another thread.

Oh yeah!

> Maybe we could print a fake log message prefix / separator before the panic though so it's treated as one message in kibana.

Great idea.